inspirehep/wsgi.py

```diff
@@ -5,7 +5,20 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
+from invenio_app.factory import app_class, create_api, config_loader, instance_path
+from invenio_base.app import create_app_factory
+from invenio_base.wsgi import create_wsgi_factory
 
-from invenio_app.factory import create_api
 
-application = create_api()
+create_app = create_app_factory(
+    "inspirehep",
+    config_loader=config_loader,
+    blueprint_entry_points=["invenio_base.api_blueprints"],
+    extension_entry_points=["invenio_base.api_apps"],
+    converter_entry_points=["invenio_base.api_converters"],
+    wsgi_factory=create_wsgi_factory({"/api": create_api}),
+    instance_path=instance_path,
+    app_class=app_class(),
+)
+
+application = create_app()
```

Comment:
I'd move this to factory.py and reuse the app in ``cli``, ``wsgi`` and ``celery``